general/clients/installing-kodi.md

```diff
@@ -1,61 +1,101 @@
-6. The first sync of the server to the local Kodi database may take some time depending on your device and library size.
-7. Once the full sync is done you can browse and play your media through Kodi, and syncs will be done periodically in the background.
-
-### Native Mode Configuration
-
-The benefit of Native mode over Add-On mode is that nothing will be transcoded, currently Add-On mode doesn't support audio with more than 5.1 channels, which means no Dolby Atmos or DTS-X. Since no transcoding happens, files play instantly.
-
-1. You need to delete and re-add your libraries on your Jellyfin server in order to set the **Optional Network Path** to your share locations if you haven't already done so previously, SMB/Samba is preferred, and **must be in the Windows format** using backslashes, *even if both the client and server are using Unix-like OSes*, for example **\\\192.168.1.7\movies**, wait for the import to finish on the server.
-2. If using a Unix-like OS you must create a local user and Samba user named **guest** (with no password) on your server and allow guest only access to your shares, if using Windows this may already be done for you by default. Alternatively, a user-password combination can be provided in the the **\\\\username:password@host\share** format.
-3. Add in your SMB/Samba shares into Kodi like you normally would when not using the Jellyfin plugin, setting the content type, but it is not necessary to import that data since this will be handled by the Jellyfin plugin.
-4. Select your libraries to add in using the Jellyfin plugin's menu, proceed to import your library.
-5. Your files should now be accessible in Kodi and function the same way as Add-On mode, if not, enable debug logging in the Jellyfin plugin in Kodi and if in a Unix-like OS, set the **log level** of Samba to 2 to see if there are issues authenticating.
+        * Note that if you have a baseurl set, you should append that value to the end of the host field.
+            * Host: `192.168.0.10:8096/jellyfin`
+    * Select user account and input password, or select "Manual Login" and fill in your user infomation
+3. Once you're succesfully authenticated with the server, you'll be asked about your preferences for this device. Among these options is the choice of Add-on vs Native modes.
 
-> [!NOTE]
-> It's recommended to install the `Kodi Sync Queue` plugin into the Jellyfin server as well.
+#### Add-on Mode
+
+Add-on mode uses the Jellyfin server to translate media files from the filesystem to Kodi.  This is the default setting for the add-on, and is sufficient for most use cases.  It will work both on the local network and over the Internet through a reverse proxy or VPN connection.  Providing network speed is sufficient, Kodi will direct play nearly all files and put little overhead on the Jellyfin server.  Exceptions to this rule are files with 7.1 audio tracks and some 4k content, which will be transcoded by the server.
```

Comment:
Does 5.1 transcode or is Kodi able to properly handle it?

Reply:
Kodi is able to play just about everything.  There's some incompatibility in the server where 7.1 audio will get transcoded down to 5.1

Reply:
Can you provide me some transcode logs? It might just be a simple profile issue

Reply:
Nope, nope I can not.  I don't have any.  That's what's been reported to me several times by other users as the reason they needed to have Native mode working

Reply:
I'm guessing the server will always max out at 5.1 for an arbitrary reason, you could probably find and remove it with no side effects.